frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -270,6 +305,9 @@ const NavBar = ({ headersData, user, setUser }: Props): ReactElement => {
         <Grid item md={6} className={searchBar ? search : searchHidden}>
           {auto()}
         </Grid>
+        <Grid item md={1} className={menu} container justifyContent="flex-end">
```

Comment:
The logic will be better if you have another boolean isAdmin then you do {isAdmin && <Button>}, the button should only render the button, not to check whether it's admin or not